pkg/applicationserver/config.go

```diff
@@ -101,7 +102,8 @@ type PubSubConfig struct {
 
 // ApplicationPackagesConfig contains application packages associations configuration.
 type ApplicationPackagesConfig struct {
```

Comment:
`ApplicationPackagesConfig` can become `Config` and move to `io/packages/config`. This way we keep the configuration in only one place and don't need to squash.

Reply:
How will this work? the `io/packages/config.Config` would have to include `io/packages.Registry`, which creates an import cycle.